Blockzilla/SettingsViewController.swift

```diff
@@ -188,26 +190,42 @@ class SettingsViewController: UIViewController, UITableViewDataSource, UITableVi
         let usageDataSubtitle = String(format: UIConstants.strings.detailTextSendUsageData, AppInfo.productName)
         let usageDataToggle = BlockerToggle(label: UIConstants.strings.labelSendAnonymousUsageData, setting: SettingsToggle.sendAnonymousUsageData, subtitle: usageDataSubtitle)
         let safariToggle = BlockerToggle(label: UIConstants.strings.toggleSafari, setting: SettingsToggle.safari)
+        let homeScreenTipsToggle = BlockerToggle(label: UIConstants.strings.toggleHomeScreenTips, setting: SettingsToggle.showHomeScreenTips)
+        
         var toggles = [Int : BlockerToggle]()
         if let biometricToggle = createBiometricLoginToggleIfAvailable() {
             toggles = [
                 1: blockFontsToggle,
                 2: biometricToggle,
                 3: usageDataToggle,
-                6: safariToggle
+                6: safariToggle,
+                7: homeScreenTipsToggle
             ]
-        }
-        else {
+            
+            if !UserDefaults.standard.bool(forKey: BrowserViewController.userDefaultsShareTrackerStatsKeyNEW) {
+                toggles[7] = nil
+            }
+        } else {
             toggles = [
                 1: blockFontsToggle,
                 2: usageDataToggle,
-                5: safariToggle
+                5: safariToggle,
+                6: homeScreenTipsToggle
             ]
+            
+            if !UserDefaults.standard.bool(forKey: BrowserViewController.userDefaultsShareTrackerStatsKeyNEW) {
+                toggles[6] = nil
+            }
         }
         if #available(iOS 12.0, *) {
             if let safariRow = toggles.first(where: { $1 == safariToggle })?.key {
                 toggles.removeValue(forKey: safariRow)
```

Comment:
You can add in the above here since we're already fetching the safariRow (so it won't add any time complexity)